### DISCUSSION TITLE
Higress Gateway 如何使用HTTPS

### DISCUSSION THREAD

**Discussion by lxyqwer** [higress-group/higress#1457] [Q&A]
`gateway:
      httpPort: 80
      httpsPort: 443
      hostNetwork: false
`
Higress Gateway 如何监听的 HTTPS 端口？通过helm部署，gateway写有httpsPort
进入higress-gateway 的 pod发现443端口并没开启，只有80端口开启了
![image](https://github.com/user-attachments/assets/25961fa0-1d11-4d5a-bbea-996b83a507ad)

**Comment by lxyqwer** [comment]
版本：gateway:1.3.0

**Comment by johnlanni** [comment]
如果没有配置https证书的域名，或域名上没有路由，是不会监听443端口的

**Reply by lxyqwer** [reply]
👌🏻，感谢
